splice/models.py

```diff
@@ -6,6 +6,16 @@
 metadata = db.metadata
 
 
+class Content(db.Model):
+    __tablename__ = "contents"
+
+    id = db.Column('id', db.Integer(), autoincrement=True, primary_key=True, info={"identity": [1, 1]})
+    name = db.Column('name', db.String(length=255), nullable=False, unique=True)
+    version = db.Column('version', db.Integer(), nullable=False)
+    created_at = db.Column('created_at', db.DateTime(), server_default=db.func.now(), nullable=False)
```

Comment:
Might want to store original content bucket/keyname here. I know it's generated in `_get_original_content`, but the key name structure may change or we may move buckets.